### ISSUE TITLE
Update chokidar to v3

### ISSUE THREAD

**Issue by polarathene** [59naga/babel-plugin-add-module-exports#87]
Presently results in a warning about breakage on node v14 installs.

```
warning gatsby > babel-plugin-add-module-exports > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning gatsby > babel-plugin-add-module-exports > chokidar > fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
```

Benefits by the maintainer are mentioned [here](https://github.com/webpack/watchpack/issues/130#issue-516540925):

> We've released Chokidar 3 in April. See my post on that: [Chokidar 3: How to save 32TB of traffic every week with one NPM package](https://paulmillr.com/posts/chokidar-3-save-32tb-of-traffic/)
> 
> The changes are pretty big. First of all, the package is 16 times smaller and uses 15 dependencies instead of 201. We've switched to n-api, which means users won't download fsevents binaries anymore. n-api is included in every nodejs installation, and it's tiny. Directory walking has been improved massively with stream implementation of readdirp. Massive RAM & CPU improvements etc.

**Comment by ljharb** [comment]
I’m not sure why chokidar is a direct optional dep at all.

**Comment by polarathene** [comment]
@ljharb it was originally in the `devDependencies` prior to 1.0.0 it seems. But even then I don't see any usage of it. `optionalDependencies` is meant to be used for allowing a package to install correctly, even if an optional dependency fails to install(I'm guessing due to those fsevent errors/warnings?).

If it's really not needed for the package, then yeah, it probably should be removed..

**Comment by ljharb** [comment]
Right, but optionalDeps is part of the API of a package - there's no such thing as optional dev deps.
